extensions/2.0/Khronos/KHR_physics_rigid_bodies/README.md

```diff
@@ -0,0 +1,509 @@
+
+| |Type|Description|
+|-|-|-|
+|**geometry**|`object`| An object describing the geometrical representation of this collider.|
+|**physicsMaterial**|`integer`|Indexes into the top-level `physicsMaterials` and describes how the collider should respond to collisions.|
+|**collisionFilter**|`integer`|Indexes into the top-level `collisionFilters` and describes a filter which determines if this collider should perform collision detection against another collider.|
+
+If the node is part of a dynamic rigid body (i.e. itself or an ascendant has `motion` properties) then the collider belongs to that rigid body and must move with it during simulation. Otherwise the collider exists as a static object in the physics simulation which can be collided with but must not be moved as a result of solving collisions or joints. A rigid body may have a compound constructed from several collision shapes by supplying multiple descendent `collider` nodes.
+
+Implementations of this extension should ensure that collider transforms are always kept in sync with node transforms - for example animated node transforms should be applied to the physics engine (even for static colliders).
+
+
+#### Geometry
+
+The `geometry` object is used to specify the shape used for collision detection and contains the following properties:
+
+| |Type|Description|
+|**shape**|`integer`| The index of a top-level `KHR_implicit_shapes.shape`, providing an implicit representation of the geometry. |
+|**node**|`integer`| The index of a glTF `node` which provides a mesh representation of the geometry. |
+|**convexHull**|`boolean`|Flag to indicate that the geometry should be a convex hull.|
+
+Exactly one of `shape` or `node` should be provided.
+
+The `shape` property indexes into the set of top-level collision shapes provided by the [KHR\_implicit\_shapes](../KHR_implicit_shapes/README.md) extension. The primitives provided by this extension allow for an implicit declaration of smooth surfaces with a minimal data footprint.
+
+Alternatively, a geometry can be specified by supplying the `node` parameter. In this scenario, the geometry should be determined by the `mesh` properties on the referenced `node`, including any child nodes. Implementations should include any mesh deformation due to use of the `skin` or `weights` properties on the `node` or the referenced `meshes`. A `geometry` object may refer to the same `node` that the `geometry` object is attached to, or it may reference any other `node`, including nodes which are not in the scene hierarchy.
+
+Collision shapes are parameterized in local space of the `node` they are associated with. If a shape is required to have an offset from the local space of the node the shape is associated with (for example a sphere _not_ centered at local origin or a rotated box,) a child node should be added with the desired offset applied, and the shape properties added to that child.
```

Comment:
This needs clarification. With the latest update where "geometry" references a node with a mesh on it, there are two nodes associated with a shape: the node a shape is defined on, and a node referenced by it. The language of "associated with" is too ambiguous.